### PR TITLE
Change text color from Highlight to HighlightText in several cases.

### LINKED ISSUE #3985: Unnecessary high contrast override is used for buttons with class ms-Link
#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: VSTS

#### Describe the issue:

In #2589, you can find many instances where the "color" property is set to "Highlight". Highlight is intended as a background color, so in a dark HC theme, it will be a dark color. 

#### Actual behavior:

The result is that many areas have a dark green text color on a black background (High Contrast Theme 1).
image

#### Expected behavior:

Perhaps the color "HighlightText" was intended? That shows up as white in HC1.
image

**Can someone confirm this expected behavior for me? If so I will submit a PR to fix. Thanks.**

### PR DESCRIPTION
**Pull request by trevorsg**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #3985 
- [x] Include a change request file using `$ npm run change`

### CODE REVIEW

**Comment by trevorsg** [comment]
@cschleiden Not sure why you got added as a reviewer... :)

**Comment by betrue-final-final** [comment]
I'm trying to check out your changes but can't get the localhost to run on Edge these days. Can you provide screenshots?

File: packages/office-ui-fabric-react/src/components/Link/Link.scss [hunk 1]
```diff
@@ -54,7 +54,7 @@ button.root {
 
   @include high-contrast {
     // Using Highlight here so we still have a themeable system color that is different from both window text and link text.
```
Comment (betrue-final-final) [inline packages/office-ui-fabric-react/src/components/Link/Link.scss]: This should actually be Hotlight

Comment (trevorsg) [reply packages/office-ui-fabric-react/src/components/Link/Link.scss]: @betrue-final-final What is Hotlight? I don't see it on [this list](https://www.sitepoint.com/css-system-styles/) of system colors.

**Comment by betrue-final-final** [comment]
It's REALLY strange that the page you looked at didn't have it. It's the link color. If you go into Windows high contrast settings, you can see all the colors they use, and these are the translations:

image

**Comment by trevorsg** [comment]
@betrue-final-final Are you sure? I hate to ask, but when I test this in Edge with HC1, Edge doesn't recognize it (red squiggle). When I try any of the others it seems to work fine.
image

**Comment by betrue-final-final** [comment]
That's odd. Maybe they changed it since Windows 8, but it was always COLOR_HOTLIGHT. Here's a snippet from aero.xml

image

Highlight in HC Black should be light blue.

**Comment by betrue-final-final** [comment]
Can you provide a screenshot of Link?

**Comment by trevorsg** [comment]
Sorry, I will have to get back to this on Tuesday. Will update at that time.

**Comment by trevorsg** [comment]
@dzearing @betrue-final-final Can someone take a second look? I simplified the PR, which is needed to fix an accessibility bug that is past SLA in VSTS.

**Pull request merged**